src/index.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.scss" >
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
Add modifiers here to see if they work properly. If your linter sees a mistake, try to place them in 1 line (so that the name of tag and all it classes will be placed in 1 line).